src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -3089,6 +3089,23 @@ public void ConstructMethodWithNullability()
             Assert.Throws<ArgumentException>(() => genericMethod.Construct(typeArguments, default));
         }
 
+        [Fact, WorkItem(65499, "https://github.com/dotnet/roslyn/issues/65499")]
+        public void NetModuleReference()
```

Comment:
Let's test scenarios when there is a mismatch between the properties that we use and what is actually in the image, and observe an error reported by the compiler.